### PR TITLE
feat: create react wrappers

### PR DESCRIPTION
**Pull request by timrombergjakobsson**
**Describe pull-request**  
Adding React wrappers. 

**Solving issue**  
Easier integration of the Tegel package in React applications.
Fixes: [CDEP-1946](https://tegel.atlassian.net/browse/CDEP-1946)

**How to test**  

1. In packages/core run `npm i`
2. Run the command `npm run build-react`, and then navigate to `tegel/packages/react/`and run `npm pack` in order to create the neccessary .tgz file. 
3. Then in your React app, drag the .tgz file into root and install it. 
4. In your App.tsx add this lines:

```jsx
import { MyComponent, defineCustomElements } from '@scania/tegel-react';

defineCustomElements();

function App() {
  return (
    <div className="App">
      <MyComponent first="Your" last="Name" />
    </div>
  );
}

export default App;


```

- Finally in your global stylesheet, add:
`@import url('@scania/tegel/dist/tegel/tegel.css');
`

[CDEP-1946]: https://tegel.atlassian.net/browse/CDEP-1946?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/react/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/scania-digital-design-system/tegel.git"
+  },
+  "author": "",
+  "license": "ISC",
```
Comment (mJarsater) [inline packages/react/package.json:24]: I think it should be MIT right?

Comment (timrombergjakobsson) [inline packages/react/package.json:24]:
```suggestion
  "license": "MIT",
```

File: packages/react/package.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
```
Comment (mJarsater) [inline packages/react/package.json:1]: Not sure where to add this but should we add a propper README to this one straight away?

File: package.json [hunk 3]
```diff
@@ -7,7 +7,8 @@
   "scripts": {
     "start": "cd packages/core && npm run storybook",
     "build": "cd packages/core && npm run build",
-    "build-angular": "npm run build && cd .. && cd angular && npm i && npm run build ",
+    "build-angular": "npm run build && cd packages/angular && npm i && npm run build ",
+    "build-react": "npm run build && cd packages/react && npm i && npm run build ",
```
Comment (mJarsater) [inline package.json:11]: Nice!

Should we add a global "build-all" script?

**Review by theJohnnyMe — APPROVED** [review]

**Review by theJohnnyMe — APPROVED** [review]

**Comment by mJarsater** [comment]
> Getting the jsxFactory error, looking into a solution.
> 
> Screenshot 2023-10-04 at 09 30 41

Never mind, me lacking the latest changes.

Pushed one change to the package-lock updating the licence. Great work!

**Review by mJarsater — APPROVED** [review]
Awesome sauce 🍝

**Review by alexandereneroth — APPROVED** [review]
⚛️

**Pull request merged**